### PR TITLE
Removed debug setting

### PR DESCRIPTION
**Pull request by t-dedah**
**Task name**: AzureResourceGroupDeploymentV2, AzureResourceManagerTemplateDeploymentV3

**Description**: Removed the debug settings for ARM deployment template tasks.

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) <Please add link to related issue here>

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/AzureResourceGroupDeploymentV2/task.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "version": {
         "Major": 2,
         "Minor": 170,
-        "Patch": 0
+        "Patch": 1
```
Comment (bishal-pdMSFT) [inline Tasks/AzureResourceGroupDeploymentV2/task.json:18]: @t-dedah please check if a package with this patch version already exists. If so, then you might have to bump up to a patch version which does not exist.
https://mseng.visualstudio.com/AzureDevOps/_packaging?_a=feed&feed=Codex-Deps

File: Tasks/AzureResourceManagerTemplateDeploymentV3/task.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
     "version": {
         "Major": 3,
         "Minor": 2,
-        "Patch": 8
+        "Patch": 9
```
Comment (bishal-pdMSFT) [inline Tasks/AzureResourceManagerTemplateDeploymentV3/task.json:18]: same here

**Review by bishal-pdMSFT — APPROVED** [review]

**Comment by bishal-pdMSFT** [comment]
@t-dedah do you want to merge to releases/m170 branch or to a fork of it? 
/cc @ddivanshu

File: Tasks/AzureResourceGroupDeploymentV2/task.json (hunk 1 above)
Comment (t-dedah) [reply Tasks/AzureResourceGroupDeploymentV2/task.json:18]: Confirmed
https://mseng.visualstudio.com/AzureDevOps/_packaging?_a=package&feed=Codex-Deps&package=Mseng.MS.TF.DistributedTask.Tasks.AzureResourceGroupDeploymentV2&protocolType=NuGet&version=2.182.0&view=versions

File: Tasks/AzureResourceManagerTemplateDeploymentV3/task.json (hunk 2 above)
Comment (t-dedah) [reply Tasks/AzureResourceManagerTemplateDeploymentV3/task.json:18]: Confirmed
https://mseng.visualstudio.com/AzureDevOps/_packaging?_a=package&feed=Codex-Deps&package=Mseng.MS.TF.DistributedTask.Tasks.AzureResourceManagerTemplateDeploymentV3&version=3.182.0&protocolType=NuGet&view=versions

**Comment by ddivanshu** [comment]
> @t-dedah do you want to merge to releases/m170 branch or to a fork of it?
> /cc @ddivanshu

Dont want to merge in it.Lets keep this branch as it is and generate nuget based upon this branch.
@t-dedah Pls update @stephenmichaelf regarding this .

**Pull request closed without merging**